daterangepicker.js

```diff
@@ -811,6 +815,16 @@
                     if (this.endDate != null && calendar[row][col] > this.startDate && calendar[row][col] < this.endDate)
                         classes.push('in-range');
 
+                    // Set custom classes configured
+                    var cellClasses;
+                    if (typeof this.dateCellClasses === 'function')
+                        cellClasses = this.dateCellClasses(calendar[row][col]);
+                    else 
+                        cellClasses = this.dateCellClasses;
+
+                    if ($.isArray(cellClasses))
```

Comment:
Rather than using array, I think it's better to use normal String classes as in [jQuery.addClass()](https://api.jquery.com/addclass/). such as `"class1 class2"`.